modules/tinymce/src/core/main/ts/keyboard/Nbsps.ts

```diff
@@ -126,13 +136,16 @@ const normalizeNbspMiddle = (text: string): string => {
   }).join('');
 };
 
-const normalizeNbspAtStart = (root: SugarElement<Node>, node: Text): boolean => {
+const normalizeNbspAtStart = (root: SugarElement<Node>, node: Text, makeNbsb: boolean): boolean => {
```

Comment:
You still have a typo here by the looks... it seems you only fixed some of them instead of all of them.